2 - ListsLogicAndIterations/Code/Dictionarys.cs

```diff
@@ -7,16 +7,31 @@ public static class Dictionarys
 	{
 		public static Dictionary<Guid, decimal> Build(Guid key, decimal value)
 		{
+            var Dict = new Dictionary<Guid, decimal>();
+            Dict.Add(key, value);
+            return Dict;
 			throw new NotImplementedException();
 		}
 
 		public static decimal GetValue(Dictionary<Guid, decimal> dict, Guid key)
 		{
+            decimal value;
+            try { value = dict[key]; }
+            catch (KeyNotFoundException)
+            {
+                value = 0;
+            } 
+            return value;
```

Comment:
The Dictionary class has methods like ContainsKey to determine this without using exceptions.